examples/distribution-gke/README.md

```diff
@@ -0,0 +1,6 @@
+# Example of chart configuration
+
+## How to install the collector on GKE
+
+In this example we will show how to install the collector on this Kubernetes
+distribution.
```

Comment:
Maybe something like that to avoid copy-pasting?

`distribution-gke-values.yaml` file should be used to install the collector in Google Kubernetes Engine (GKE).

Reply:
Ok sounds good